Makefile

```diff
@@ -46,8 +46,8 @@ RELEASE_OUT ?=
 TELEPORT_TAG = 3.2.16
 # TELEPORT_REPOTAG adapts TELEPORT_TAG to the teleport tagging scheme
 TELEPORT_REPOTAG := v$(TELEPORT_TAG)
-PLANET_TAG := 7.0.51-$(K8S_VER_SUFFIX)
-PLANET_BRANCH := $(PLANET_TAG)
+PLANET_TAG := 7.0.52-$(K8S_VER_SUFFIX)
+PLANET_BRANCH := sergei/70x/bump_helm2
```

Comment:
This seems incorrect. Changing the PLANET_TAG while at the same time hard coding a branch name.

Reply:
@lenko-d Devs sometimes do this so that robotest will test the correct branch before merging the dependent branch (planet in this case), and change it back before merging the main repo.